### DISCUSSION TITLE
Nonce fix for FEVM testnet slow/hanging transactions

### DISCUSSION THREAD

**Discussion by trruckerfling** [filecoin-project/community#609] [Developers]
Hanging transactions with Filecoin testnet? This is usually due to a network upgrade or reset, where your account nonce becomes out of sync with the current network state. Fixable! Try:
- [Reset your Metamask account](https://metamask.zendesk.com/hc/en-us/articles/360015488891-How-to-reset-an-account) and it should get the nonce from the node so you'll be back in sync.
- For addresses that don't ever send messages, set the nonce to 0

**Comment by trruckerfling** [comment]
Example of error:
```
error: ProviderError: failed to estimate gas: message execution failed: exit 33, revert reason: none, vm error: message failed with backtrace:
  00: f021524 (method 3844450837) -- contract reverted (33)
```

Fix, [OP](https://filecoinproject.slack.com/archives/CRK2LKYHW/p1678701275601249?thread_ts=1678637038.372029&cid=CRK2LKYHW) by @ottpeter 

```
const options = {
          gasLimit: 1000000000,
          //maxPriorityFeePerGas: 2000000000
        }

        
        //send a transaction to call claim_bounty() method
        transaction = await dealRewarder.claim_bounty(dealid, options)
```

**Comment by longfeiWan9** [comment]
@trruckerfling I updated the title and content to remove Hyperspace. Let's use testnet which is more generic, and less confusing.
